build.bat

```diff
@@ -1 +1 @@
-msbuild .\projects\openrct2.sln /p:Configuration=Release /p:Platform=Win32
\ No newline at end of file
+msbuild .\projects\openrct2.sln /p:Configuration=%CONFIGURATION% /p:Platform=Win32

```

Comment:
This is deliberate for my own use case. The VS command prompt by default uses debug configuration and I wanted to override that.